wandb/cli/cli.py

```diff
@@ -1519,6 +1519,351 @@ def scheduler(
         raise e
 
 
+@cli.group("job")
+def job() -> None:
+    pass
```

Comment:
What is this?

Reply:
this is the CLI job group, enabling the `wandb job x` paradigm. If you just run `wandb job` it should print out a nice help message. @TimH98